experimental/table/client/src/main/java/alluxio/client/catalog/CatalogMasterClient.java

```diff
@@ -12,11 +12,17 @@
 package alluxio.client.catalog;
 
 import alluxio.Client;
+import alluxio.exception.status.AlluxioStatusException;
 import alluxio.master.MasterClientContext;
 
+import org.apache.hadoop.hive.metastore.api.ColumnStatisticsObj;
```

Comment:
is importing data structures from Hive Metastore a workaround or final design?

Reply:
The importing data structures from Hive metastore is a workaround to simply implementation. We expect we have our data structure in the final design.